connector/connectorRunner.go

```diff
@@ -102,10 +97,6 @@ func (cr *connectorRunner) Run() error {
 	if err != nil {
 		log.Error(err.Error())
 	}
```

Comment:
please make sure that the components close() is called in reverse order from their start

Reply:
good catch. changed the order.